adapters/Pangle/Public/Headers/GADMediationAdapterPangle.h

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+typedef NS_ENUM(NSInteger, GADPangleErrorCode) {
+    /// Missing server parameters.
+    GADPangleErrorInvalidServerParameters = 101,
+    /// Banner size mismatch.
+    GADPangleErrorBannerSizeMismatch = 102,
+};
+
+@interface GADMediationAdapterPangle : NSObject <GADRTBAdapter>
+
+/// Set the COPPA setting in Pangle SDK.
+///
+/// @param COPPA An integer value that indicates whether the app should be treated as
+/// child-directed for purposes of the COPPA.  0 means false. 1 means true. -1 means
+/// unspecified. Any value outside of -1, 0, or 1 will result in this method being a no-op.
++ (void)setCOPPA:(NSInteger)COPPA;
```

Comment:
I actually still prefer not exposing this API to the publishers because they can just simply set the coppa value by following this guide: https://developers.google.com/admob/ios/targeting, and the below edge case will certainly cause an issue.

If the publishers add `[GADMediationAdapterPangle setCoppa:1];` but forget to do the following (as in [this guide](https://developers.google.com/admob/ios/targeting)):
```
[GADMobileAds.sharedInstance.requestConfiguration tagForChildDirectedTreatment:YES];
```

The original `coppa` value set by this API will be `1`, but then the implementation below in each `load<format>AdForAdConfiguration` method will overwrite the `coppa` value in Pangle SDK and set it to `-1`.
```
[GADMediationAdapterPangle setCOPPA:(adConfiguration.childDirectedTreatment ? adConfiguration.childDirectedTreatment.integerValue : -1)];
```

We can add more logics to prevent this but this will certainly make the code more complicated and error prone.

I would prefer have the source of truth only comes from `adConfiguration.childDirectedTreatment`.

Let me know if that makes sense to you.